include/mull/AST/ASTTypes.h

```diff
@@ -0,0 +1,40 @@
+//
+// Created by Stanislav Pankevich on 2019-10-18.
+//
+
+#pragma once
+
+#include <clang/AST/Stmt.h>
+
+#include <string>
+#include <unordered_map>
+
+namespace mull {
+enum TraverseMask: int {
```

Comment:
I didn't find a better way of making it parameterize-able which mutations AST Search should look for. Without this functionality it is not possible to get just one mutation in certain test cases of the `ASTVisitor_*_Tests`.